ozark/src/main/java/org/glassfish/ozark/bootstrap/OzarkInitializer.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright © 2017 Ivar Grimstad (ivar.grimstad@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.glassfish.ozark.bootstrap;
+
+import org.glassfish.ozark.core.ViewResponseFilter;
+import org.glassfish.ozark.util.AnnotationUtils;
+
+import javax.mvc.annotation.Controller;
+import javax.ws.rs.core.Configuration;
+import javax.ws.rs.core.FeatureContext;
+import java.util.Arrays;
+import java.util.ServiceLoader;
+import java.util.logging.Logger;
+
+/**
+ * Main class for triggering initialization of Ozark
+ *
+ * @author Christian Kaltepoth
+ */
+public class OzarkInitializer {
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.